src/commands/asset/unlink.rb

```diff
@@ -3,19 +3,23 @@
 module Metalware
   module Commands
     module Asset
-      class Unlink < Metalware::CommandHelpers::BaseCommand
-        include CommandHelpers::AssetHelper
-
+      class Unlink < CommandHelpers::BaseCommand
         private
 
-        attr_reader :node_name
+        attr_reader :node_name, :cache
 
         def setup
           @node_name = args[0]
+          @cache = Cache::Asset.new
         end
 
         def run
-          unassign_node_from_cache(node_name)
+          unassign_node_from_cache
+        end
+
+        def unassign_node_from_cache
+          cache.unassign_node(node_name)
```

Comment:
This can use the `cache.update` method

Reply:
This was not based on that branch but once this PR is merged the next thing on the list is to update, no pun intended, everywhere that could use the `cache.update` method